dracut/anaconda-lib.sh

```diff
@@ -230,6 +270,27 @@ online_netdevs() {
     done
 }
 
+# Filter locations that are http, https or ftp urls.
+get_urls() {
+    local locations="${1}"
+    local location
+
+    # No locations.
+    [ -z "$locations" ] && return
```

Comment:
I guess you can remove this. When you have the `for` cycle below it will be skipped there without an additional code.

Reply:
Fixed.